packages/react-core/src/components/Page/Page.tsx

```diff
@@ -27,6 +27,8 @@ export interface PageProps extends React.HTMLProps<HTMLDivElement> {
   notificationDrawer?: React.ReactNode;
   /** Flag indicating Notification drawer in expanded */
   isNotificationDrawerExpanded?: boolean;
+  /** Sets drawer width, in either pixels or percentage. */
+  drawerWidth?: string;
```

Comment:
You can also set `minSize` and `maxSize` - should we expose those, too? In the slack thread referenced in the issue, the user dug up the var to set for `minSize`, so seems like those would be useful for users. https://github.com/patternfly/patternfly-react/blob/0c270bc245f50ed4e990918441903ccd6686044c/packages/react-core/src/components/Drawer/DrawerPanelContent.tsx#L317-L325

If so they could be different props, or we could support a string and an object here - if a string, it's used for `defaultSize` as was added in this PR. Otherwise an object could have `min`, `default`, and `max` keys that could be used to set any/all of those props on the drawer.